packages/edit-site/src/components/page-pages/index.js

```diff
@@ -315,11 +338,14 @@ export default function PagePages() {
 					paginationInfo={ paginationInfo }
 					fields={ fields }
 					actions={ actions }
+					bulkActions={ bulkActions }
 					data={ pages || EMPTY_ARRAY }
 					getItemId={ ( item ) => item.id }
 					isLoading={ isLoadingPages || isLoadingAuthors }
 					view={ view }
 					onChangeView={ onChangeView }
+					selection={ selection }
```

Comment:
Haven't tested anything, but it seems weird for the consumer to handle the selection state. What's the rationale here?

Reply:
It was already like that, I'm not adding the selection array to the state, it already existed and was part of the state of the consumer, so I opted to keep it as it was. I think the reason was that selection is used for the side-by-side view which is a different view type. I can look into changing this if you prefer to have it inside dataviews.

Reply:
Oh.. This selection is part of the pages for the side by side then. For data views selection functionality we should keep the selection state internally and not pass it like here.

Reply:
I can see the container wanting to control the selection: 

 - For instance to adapt the url
 - to show a child view
 - ...

If we do "uncontrolled", we'll be forced to have two behaviors for the same thing. and we know that we want controlled selection anyway.

Reply:
I don't disagree with that, but should the handling of the state be the consumer's responsibility? Should we do it differently? Maybe use store selectors?

Reply:
Store could work if we had a dedicated store for data views. I think we should make the data views component dependent on edit site store. But having the consumer hold a state for the selection doesn't sound bad to me to be honest.

Reply:
We can start in edit site store(locked) and soon that we will have dataviews in its own package, we move them there.

Reply:
I'm not sure I understand why we want to change that.

Reply:
For me it feels something that should be handled internally, but okay, let's do it the consumer. I guess we can re evaluate if needed in the future.